dashboard/app/models/lesson.rb

```diff
@@ -267,6 +267,14 @@ def lesson_plan_base_url
     CDO.code_org_url "/curriculum/#{script.name}/#{relative_position}"
   end
 
+  def unit_standards_url
```

Comment:
For both of these, I don't know how to confirm that these pages do indeed exist for the curriculum that the teacher is on NOR how to ensure that these are the right international urls. 

It looks like this is already inconsistent internationally - for example this course: https://studio.code.org/s/csd1-2021 has `https://studio.code.org/s/csd1-2021/normas` listed as the URL for standards when viewing the page in Spanish but that gives you a sad bee.  If you manually change this to `https://studio.code.org/s/csd1-2021/standards` you get the standards in Spanish...

Reply:
@mgc1194 do you have any insight on the localization question here?

Reply:
we generally should not translate a url  part like `standards`. the link pointing to `normas` sounds like a bug, and the URL in your code seems like the right one to use. 

to get the correct url, you should use rails path helpers: https://guides.rubyonrails.org/routing.html#path-and-url-helpers in this case `standards_script_path`.

Reply:
I tried this for a course which has no lesson standards, and the page seems to render fine there: https://studio.code.org/s/mc/standards

one additional thought here is whether you'd want to exclude this entry for units with no standards. you can take a peek here to see how you might check whether there are any standards to show: https://github.com/code-dot-org/code-dot-org/blob/f29ccef4e4e7a1eba384fd952d32b747ddd64167/dashboard/app/controllers/scripts_controller.rb#L220-L221

Reply:
thanks for switching to path helpers. here is a jira ticket to track the `normas` issue: https://codedotorg.atlassian.net/browse/TEACH-1380